src/origyn_nft_reference/main.mo

```diff
@@ -102,7 +102,7 @@ shared (deployer) actor class Nft_Canister(__initargs : Types.InitArgs) = this {
 
```

Comment:
line 21: We should import libs as "mo:map_7_0_0/Map" only in migrations folder, across the project we should use "mo:map/Map" which should always be the latest version. We need this to avoid changing import names after each migration

Reply:
Ok...thanks!